package.json

```diff
@@ -37,6 +37,7 @@
     "grunt-contrib-uglify": "0.8.0",
     "grunt-mocha-test": "0.12.7",
     "grunt-shell-spawn": "0.3.1",
+    "mocha": ">= 1.20.0",
```

Comment:
As I said on clamshell, I'd prefer we just stick to our policy of locking things down for now.